content/test/integration/E2ETestEnvironment.ts

```diff
@@ -49,7 +49,7 @@ export class E2ETestEnvironment {
       })
     })
     this.database = await createDatabaseComponent({ logs, env: this.sharedEnv, metrics })
-    this.database.start()
+    if (this.database.start) this.database.start()
```

Comment:
is this correct? Shouldn't it be the other way out?

Reply:
The `database` now implements the WKC `IBaseComponent` interface where `start` and `stop` are optional methods. I don't think there's another better way out to do that.